src/app/payment/PaymentForm.tsx

```diff
@@ -139,6 +139,8 @@ const PaymentForm: FunctionComponent<PaymentFormProps & FormikProps<PaymentFormV
                 values={ values }
             />
 
+            <div className="fieldsContainer" id="checkout-payment-fields" />
```

Comment:
per my comment here https://github.com/bigcommerce/checkout-sdk-js/pull/1105#discussion_r616808766, there is no need to create a separate container in the payment form, each payment method renderer can handle this part next to radio-buttons, with such widget as https://github.com/bigcommerce/checkout-js/blob/051363765fb15c03d818dcb09d0668be574f490a/src/app/payment/paymentMethod/HostedWidgetPaymentMethod.tsx#L269